openwisp_monitoring/monitoring/migrations/__init__.py

```diff
@@ -37,6 +37,38 @@ def assign_permissions_to_groups(apps, schema_editor):
             )
 
 
+def assign_inline_permissions_to_groups(apps, schema_editor):
+    create_default_permissions(apps, schema_editor)
+    operators_read_only_admins_manage = [
+        'check',
```

Comment:
@nemesisdesign should we move the data migration for the `Check` model to the `check` sub-app or is it okay here?

Reply:
this one makes more sense to have it in the checks app